src/countries.ts

```diff
@@ -769,1731 +771,2734 @@ export interface Country {
-    numeric: '796',
-    currencyCodes: ['USD'],
-  },
-  {
-    name: 'Tuvalu',
-    alpha2: 'TV',
-    alpha3: 'TUV',
-    numeric: '798',
-    currencyCodes: ['AUD'],
-  },
-  {
-    name: 'Uganda',
-    alpha2: 'UG',
-    alpha3: 'UGA',
-    numeric: '800',
-    currencyCodes: ['UGX'],
-  },
-  {
-    name: 'Ukraine',
-    alpha2: 'UA',
-    alpha3: 'UKR',
-    numeric: '804',
-    currencyCodes: ['UAH'],
-  },
-  {
-    name: 'United Arab Emirates',
-    alpha2: 'AE',
-    alpha3: 'ARE',
-    numeric: '784',
-    currencyCodes: ['AED'],
-  },
-  {
-    name: 'UK',
-    alpha2: 'GB',
-    alpha3: 'GBR',
-    numeric: '826',
-    currencyCodes: ['GBP'],
-  },
-  {
-    name: 'Tanzania',
-    alpha2: 'TZ',
-    alpha3: 'TZA',
-    numeric: '834',
-    currencyCodes: ['TZS'],
-  },
-  {
-    name: 'U.S. Outlying Islands',
-    alpha2: 'UM',
-    alpha3: 'UMI',
-    numeric: '581',
-    currencyCodes: ['USD'],
-  },
-  {
-    name: 'U.S. Virgin Islands',
-    alpha2: 'VI',
-    alpha3: 'VIR',
-    numeric: '850',
-    currencyCodes: ['USD'],
-  },
-  {
-    name: 'US',
-    alpha2: 'US',
-    alpha3: 'USA',
-    numeric: '840',
-    currencyCodes: ['USD'],
-  },
-  {
-    name: 'Uruguay',
-    alpha2: 'UY',
-    alpha3: 'URY',
-    numeric: '858',
-    currencyCodes: ['UYU'],
-  },
-  {
-    name: 'Uzbekistan',
-    alpha2: 'UZ',
-    alpha3: 'UZB',
-    numeric: '860',
-    currencyCodes: ['UZS'],
-  },
-  {
-    name: 'Vanuatu',
-    alpha2: 'VU',
-    alpha3: 'VUT',
-    numeric: '548',
-    currencyCodes: ['VUV'],
-  },
-  {
-    name: 'Venezuela',
-    alpha2: 'VE',
-    alpha3: 'VEN',
-    numeric: '862',
-    currencyCodes: ['VES'],
-  },
-  {
-    name: 'Vietnam',
-    alpha2: 'VN',
-    alpha3: 'VNM',
-    numeric: '704',
-    currencyCodes: ['VND'],
-  },
-  {
-    name: 'Wallis & Futuna',
-    alpha2: 'WF',
-    alpha3: 'WLF',
-    numeric: '876',
-    currencyCodes: ['XPF'],
-  },
-  {
-    name: 'Western Sahara',
-    alpha2: 'EH',
-    alpha3: 'ESH',
-    numeric: '732',
-    currencyCodes: ['MAD'],
-  },
-  {
-    name: 'Yemen',
-    alpha2: 'YE',
-    alpha3: 'YEM',
-    numeric: '887',
-    currencyCodes: ['YER'],
-  },
-  {
-    name: 'Zambia',
-    alpha2: 'ZM',
-    alpha3: 'ZMB',
-    numeric: '894',
-    currencyCodes: ['ZMW'],
-  },
-  {
-    name: 'Zimbabwe',
-    alpha2: 'ZW',
-    alpha3: 'ZWE',
-    numeric: '716',
-    currencyCodes: ['ZWL'],
-  },
-  {
-    name: '\u00C5land Islands',
-    alpha2: 'AX',
-    alpha3: 'ALA',
-    numeric: '248',
-    currencyCodes: ['EUR'],
-  },
-  {
-    name: 'Palestine, State of',
-    alpha2: 'PS',
-    alpha3: 'PSE',
-    numeric: '275',
-    currencyCodes: ['USD'],
+    previousCreatedOn: '2018-01-01T00:00:00.000Z',
+    createdOn: '2023-01-01T00:00:00.000Z',
+  },
+  {
+    type: 'create',
+    item: {
+      name: 'Croatia',
+      alpha2: 'HR',
+      alpha3: 'HRV',
+      numeric: '191',
+      currencyCodes: ['EUR'],
+    },
+    createdOn: '2023-01-01T00:00:00.000Z',
+  },
```

Comment:
we should sort the data in the code based on the createdOn. that's how it will be handled in the future with new changes.